modules/configuring-queue-proxy.adoc

```diff
@@ -0,0 +1,68 @@
+:_content-type: PROCEDURE
+[id="serverless-configuring-queue-proxy_{context}"]
+= Configuring Queue Proxy Resources for a Knative Service
+
+Apart from configuring the Queue Proxy resource requests and limits globally in the deployment configmap, you can set them at the service level using the corresponding annotations targeting CPU, memory, and ephemeral storage resource types.
+
+.Prerequisites
+
+* {pipelines-title} must be installed on your cluster.
+
+* You have installed the OpenShift (`oc`) CLI.
+
+* You have installed the Knative (`kn`) CLI.
+
+.Procedure
+
+* Modify the configmap of your service with resource requests and limits:
++
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: example-service
+  namespace: default
+  annotations:
+    queue.sidecar.serving.knative.dev/cpu-resource-request: "1"
+    queue.sidecar.serving.knative.dev/cpu-resource-limit: "2"
+    queue.sidecar.serving.knative.dev/memory-resource-request: "1Gi"
+    queue.sidecar.serving.knative.dev/memory-resource-limit: "2Gi"
+    queue.sidecar.serving.knative.dev/ephemeral-storage-resource-request: "400Mi"
+    queue.sidecar.serving.knative.dev/ephemeral-storage-resource-limit: "450Mi"
+----
++
+Alternatively, you can use the special `queue.sidecar.serving.knative.dev/resource-percentage` annotation, which calculates the Queue Proxy resources as a percentage of the application container. In this case, the following minimum and maximum boundaries are applied to the CPU and memory resource requirements:
++
```

Comment:
OK. I found the boundaries [in code](https://github.com/openshift-knative/serving/blob/release-v1.11/pkg/reconciler/revision/resources/resourceboundary.go#L30). It looks correct.

Reply:
Maybe I'd add one more sentence to make it more clear how the boundaries are applied. It was a little confusing to me the first time I read it. Something like this: 
`When CPU and memory resource requirements are calculated from the application container requirements and they're outside the boundaries below, the values are adjusted to fit within the boundaries.`